static/js/server_events_dispatch.js

```diff
@@ -120,13 +119,6 @@ export function dispatch_normal_event(event) {
             }
             break;
 
-        case "hotspots":
-            hotspots.load_new(event.hotspots);
-            page_params.hotspots = page_params.hotspots
-                ? page_params.hotspots.concat(event.hotspots)
-                : event.hotspots;
-            break;
-
```

Comment:
We definitely need the `server_events` handler here -- otherwise if you have two browser windows open, the other will have stale state in an annoying way.